.github/ISSUE_TEMPLATE/NEW-IMPLEMENTATION.yml

```diff
@@ -0,0 +1,55 @@
+name: "Interest in Implementing - New Multibase Library or System"
+description: Express interest in possible new multibase library or system
```

Comment:
From the README:

> If you'd like to switch a project over to multibase, whether by creating a new multibase implementation or building on one of those listed above, please file an issue in this repository using the "Interested in implementing" issue template.

This isn't really actionable and I'm unsure what we're supposed to do with these issues. Just a _nod_ and and ignore? I'd like these to be actionable and not leading to cruft in the issue tracker since we already have a tendency toward cruft-buildup.

I _think_ the purpose here might be to get a new implementation/library/system onto a list somewhere, can we just shunt people to open a pull request to add their thing? The "express interest" only really tells me that the issue opener is going to open a new issue telling us what they have in their head, which may be awesome, or may be terrible, but it's not really actionable without a _thing_ or an explicit question--which is fine, but this isn't a template for a question.

Reply:
Would this be better handled by a GH "Discussion"? TBH I wanted an "expressed interest" tracker somewhere for IETF purposes, because the perennial question of standardization is "for whom is this actually useful outside of the PLN/IPFS community that spawned it?"...

Reply:
In the interest of not holding up this PR, I'll just cut that template for now and open a new PR adding it as a Discussion template (which I was today years old when I realized was a possibility)